package.json

```diff
@@ -74,6 +74,7 @@
   "scripts": {
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
+    "dev": "grunt watch",
```

Comment:
I was mostly checking to make sure this was intentional and I'm satisfied that it is. Thanks!